ReportUtilityBills/measure.rb

```diff
@@ -677,8 +677,9 @@ def setup_utility_outputs()
   def get_outputs(fuels, utility_bill_scenario)
     fuels.each do |(fuel_type, _is_production), fuel|
       unit_conv = UnitConversions.convert(1.0, 'J', fuel.units)
-      unit_conv /= 139.0 if fuel_type == FT::Oil
-      unit_conv /= 91.6 if fuel_type == FT::Propane
+      unit_conv = UnitConversions.convert(unit_conv, fuel.units, 'gal_fuel_oil') if fuel_type == FT::Oil # Convert kBtu to gallons of oil
+      unit_conv = UnitConversions.convert(unit_conv, fuel.units, 'gal_propane') if fuel_type == FT::Propane # Convert kBtu to gallons of propane
+      unit_conv = UnitConversions.convert(unit_conv, fuel.units, 'mbtu') if [FT::Coal, FT::WoodCord, FT::WoodPellets].include? fuel_type # Convert kBtu to MBtu
```

Comment:
This change doesn't seem right. Why are you converting to `$/MBtu`? We were previously reporting `$/kBtu`, which is [consistent with the documentation](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#fuel-rates).

Reply:
I must have been mistaken. I guess I should just delete it.